website/docs/installation/prompt.mdx

```diff
@@ -32,7 +32,7 @@ oh-my-posh get shell
 Edit your PowerShell profile script, you can find its location under the `$PROFILE` variable in your preferred PowerShell version. For example, using notepad:
 
 ```powershell
-notepad $PROFILE
+mkdir ~/Documents/PowerShell/ | notepad $PROFILE
```

Comment:
this needs to be an info part and cross platform too. This will only work on Windows and these docs are cross platform.

:::info
When the above command gives an error, make sure you create the profile first.

```powershell
New-Item –Path $Profile –Type File –Force
```
:::

Reply:
Totally agree.